.jenkins/cscs/env-gcc-cuda-11.sh

```diff
@@ -4,17 +4,21 @@
 # Distributed under the Boost Software License, Version 1.0. (See accompanying
 # file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 
-cxx_std="17"
+cxx_std="20"
+cuda_std="17"
 gcc_version="9.3.0"
 boost_version="1.75.0"
 hwloc_version="2.0.3"
 cuda_version="11.2.0"
+fmt_version="9.1.0"
```

Comment:
Thanks. During compilation of which file does that happen?

We definitely have that covered here: https://github.com/pika-org/pika/blob/41b4abb929879ab715759c4fd78302f48ae1b53f/libs/pika/logging/include/pika/logging/level.hpp#L61-L69. But this may be one (of many) problems that we get from allowing different standards. I'd like to see how badly other things break from this as well before we decide what to do here...